### PR TITLE
[xbmc][depends][win32] Fix python that broke in #12502

### PR DESCRIPTION
**Pull request by Paxxi**
Missed some build file changes for the new package format
python cmake file packaged the wrong architecture for extensions

## Types of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the [Code guidelines](https://codedocs.xyz/xbmc/xbmc/code_guidelines.html) of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [ ] I have read the [CONTRIBUTING](https://github.com/xbmc/xbmc/blob/master/CONTRIBUTING.md) document
- [ ] I have added tests to cover my change
- [ ] All new and existing tests passed

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -246,6 +246,7 @@ lib/cpluff/stamp-h1
 /project/BuildDependencies/build
 /project/BuildDependencies/scripts/tmp
 /project/BuildDependencies/src
+/project/BuildDependencies/bin/python
```
Comment (Rechi) [inline .gitignore]: why?

**Review by Rechi — APPROVED** [review]
everything expect the gitignore change looks good

File: .gitignore (hunk 1 above)
Comment (Paxxi) [inline .gitignore]: Oh, I made that change it seems before the package change. Will fix it

**Comment by Paxxi** [comment]
win64 is test suite crashing.

**Pull request merged**